Build header section for mobile:

NoelLincoln opened the pull request:
add a container div and set display property to flex
    add a div for logo and mobile menu inside the container
    design the logo and menu section using figma
    add menu icon svg and logo svg files to assets directory

NoelLincoln committed:
Build header section for mobile:
    add a container div and set display property to flex
    add a div for logo and mobile menu inside the container
    design the logo and menu section using figma
    add menu icon svg and logo svg files to assets directory

NoelLincoln committed:
Fix on font error:
    correct misplaced double quote that brought up an error

NoelLincoln commented:
I designed the logo and mobile menu section and focused on using flexbox to align the divs

NoelLincoln committed:
Fix:
    Add alt tag that was missing in the logo

ademibowale reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @NoelLincoln  , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._ 
- [ ] I try to reach out to you on slack but i couldnt get you please fix this issues and resumit your project.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln committed:
Add headline section :
    Add a container div for the heading tag and introduction text
    Add new logo and styling for mobile menu navigation
    Add media query for mobile devices and desktops
    Rearrange items in flex container and remove unnecessary divs

NoelLincoln committed:
Fix social media icons and headline background
    use individual vectors for each icon and add styling
    reduce logo size
    set background in headline section to cover and no-repeat

NoelLincoln committed:
Fix social media icons and headline background
    use individual vectors for each icon and add styling
    reduce logo size
    set background in headline section to cover and no-repeat

NoelLincoln commented on index.html:
```diff
@@ -5,11 +5,32 @@
+          <div class="menu-icons">
+            <div class="horizontal-bars">
+              <a href="">
+                <img src="assets/images/mobile-menu.svg" alt="" />
```
Hello @ademibowale ,
I made the necessary changes. Kindly check.

NoelLincoln committed:
Add text to headline section header

NoelLincoln commented:
Hello @ademibowale ,
I have made the requested changes and everything is fine. Kindly check and let me know since I have to move to the next part.
Thank you.

AbdusSattar-70 commented on index.html:
```diff
@@ -5,11 +5,53 @@
-    <header class="main-header">
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <nav class="navbar">
+      <!-- <div class="toolbar"> -->
+      <div class="logo">
+        <a href="">
+          <img src="assets/images/logo.png" alt="my awesome logo" />
+        </a>
+      </div>
+      <div class="tertiary-enabled">
+        <img
+          src="assets/images/mobile-menu.svg"
+          alt="menu icon"
+          class="mobilenav"
+        />
+        <!-- </div> -->
+      </div>
+    </nav>
```
- [ ] I appreciate🎉🎉 your efforts in incorporating the `<nav>` tag into your project. To adhere to HTML5 structural guidelines, it is recommended to place the `<nav>` element inside the `<header>` tag. 
- [ ] This ensures a more semantic and organized structure for your HTML document. By following this practice, you can enhance the overall code structure and maintain compliance with HTML5 standards.🤔🤔🤔
- [ ] For more information please [check it](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure)
- [ ] Please use logos width and height so that it will fix the below picture issue 
- [ ] Your decision 
![Screenshot 2023-06-03 143457](https://github.com/NoelLincoln/mportfolio/assets/118761303/a4f3cf09-97fa-42ff-997b-8988c3ebaa98)

AbdusSattar-70 commented on style.css:
```diff
@@ -0,0 +1,246 @@
+
+/* media query for mobile devices */
+@media only screen and (max-width: 375px) {
+  body {
+    /* header-background-mobile */
+
+    position: absolute;
+    width: 375px;
+    height: 718px;
+    background: #fff;
```
- [x] I kindly 🙏🙏request that you ensure your project is responsive according to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), 
- [x] specifically targeting mobile versions ranging from `375px to 768px.` By optimizing the design for different screen sizes, you will create a more user-friendly and visually appealing experience for your audience. 🥱🥱
- [x] Your attention to this detail will greatly enhance the overall quality of your project.
![res](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/35405c2a-a681-43e3-bb83-a31439f33517)

AbdusSattar-70 reviewed (changes requested):
## REQUIRED CHANGES  ♻️ 
Hi @NoelLincoln , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- PROFESSIONAL README
- NO LINTER ERROR
- FOLLOW GITHUB WORKFLOWS
![AlmostGIF](https://github.com/NoelLincoln/mportfolio/assets/118761303/e928691c-a510-4e83-99f5-25fbc05e05cf)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln committed:
FIx on responsiveness:
    Change px values to percentages to allow for responsiveness
    Use media query to add breakpoints for mobile and desktop screens
    Place nav inside header tag
    use logos width and height to remove overflow

NoelLincoln committed:
Fix linter errors on css file

NoelLincoln committed:
remove inline css on logo image

NoelLincoln commented:
Hello @AbdusSattar-70 ,
I've made the changes: made the page responsive and added breakpoints for mobile and desktop screens

AbelG101 reviewed (commented):
![](https://media.giphy.com/media/PkFYci8BUkmdA7K1qa/giphy.gif)

# Hi @NoelLincoln 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Since you're repeating the block with a previously unapproved PR please create a new PR and submit the link so we can review it for you as a first review. ✔

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
